Unknown CMake command "find_package_handle_standard_args"

Trying to build project with enabled examples:
```bash
$ mkdir build && cd build
$ cmake -DPISTACHE_BUILD_EXAMPLES=ON ..
```
and getting following error:
```
-- The C compiler identification is GNU 7.4.0
-- The CXX compiler identification is GNU 7.4.0
-- Check for working C compiler: /usr/bin/cc
-- Check for working C compiler: /usr/bin/cc -- works
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Detecting C compile features
-- Detecting C compile features - done
-- Check for working CXX compiler: /usr/bin/c++
-- Check for working CXX compiler: /usr/bin/c++ -- works
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Error at CMakeModules/FindRapidJSON.cmake:54 (find_package_handle_standard_args):
  Unknown CMake command "find_package_handle_standard_args".
Call Stack (most recent call first):
  examples/CMakeLists.txt:18 (find_package)


-- Configuring incomplete, errors occurred!
See also "/home/user/src/pistache/build/CMakeFiles/CMakeOutput.log".
```
It seems that include of `FindPackageHandleStandardArgs` module is missing within `FindRapidJSON.cmake`.

--- a/CMakeModules/FindRapidJSON.cmake
+++ b/CMakeModules/FindRapidJSON.cmake
@@ -51,6 +51,7 @@ if (NOT RAPIDJSON_FOUND)
   ##____________________________________________________________________________
   ## Actions taken when all components have been found
 
+  include (FindPackageHandleStandardArgs)
   find_package_handle_standard_args (RAPIDJSON DEFAULT_MSG RAPIDJSON_INCLUDES)
 
   if (RAPIDJSON_FOUND)

